CONTRIBUTING.md

```diff
@@ -0,0 +1,291 @@
+- `revert:` Reverts a previous commit
+- `perf:` A code change that improves performance
+
+---
+
+### How to write and structure your code
+
+To ensure a consistent and enjoyable coding experience for everyone, we have established guidelines for writing our code.
+
+#### Styling with CSS Modules
+
+We use CSS modules to style our components. This prevents naming conflicts by adding a unique prefix to all components.
+A CSS module file is created by adding `.module.css` to the end of the CSS file.
+
+#### Use of design tokens
+
+When styling our components we try to always use semantic tokens from the `@digdir/design-system-tokens` package when available.
+Using hard-coded values is not reusable and we therefore try to avoid this.
+Take a look at how other components are styled to see examples.
+
+#### Code formatting with Prettier
+
+In this project, we employ [Prettier](https://prettier.io/) for code formatting. It is advisable to configure your code editor to automatically format files upon saving. This practice will prove beneficial when merging your changes into the main branch. It's worth noting that we enforce rigorous code checks in pull requests, emphasizing the importance of consistent code formatting.
+
+#### Linting with ESLint and Stylelint
+
+TypeScript and CSS files have been configured with linting, which means that the project will scan these files for potential problems or issues. Linting helps maintain code quality by detecting errors, enforcing coding conventions, and promoting best practices. You have to fix all errors and warnings before the code can be merged into the main branch.
+
+#### Use of TypeScript files
+
+In code contributions for this project, we do not permit JavaScript files. The use of TypeScript ensures the safety and testability of our code.
+
+---
+
+## Publishing NPM packages
+
+The following documentation outlines the process for releasing new versions of the NPM packages. Please note that in order to release, you must have an NPM account that is connected to the Digdir organization on NPM. Make sure you are in the `main` branch before proceeding further. Publishing from other branches may lead to issues with the changelog.
+
+### 1. Build distribution files
+
+`yarn build`
+
+Build distribution files for all the packages. Make sure they all run successfully before proceeding to next step.
+
+### 2. Prepare new version
+
+`yarn lerna:version`
+
+This step does a few things:
+
+- Suggests a new version based on the latest commits. Make sure the version is correct before clicking enter. A user error with a commit message might suggest a version that is wrong.
+- Creates a new tag with the latest version number.
+- Commits the changes.
+- Pushes the changes to github.
+
+### 3. Publish to NPM
+
+`yarn lerna:publish`
+
+Make sure you are logged in to your NPM account from the terminal you are trying to publish from.
+
+`npm whoami` wil check if you are logged in.
```

Comment:
I would move this above the `lerne:publish` command, to ensure it is checked. Possibly make it a separate step.